openstf/pipeline/create_basecase_forecast.py

```diff
@@ -73,21 +56,21 @@ def create_basecase_forecast_pipeline(
         ],  # Generate features for load 7 days ago and load 14 days ago these are the same as the basecase forecast.
     ).add_features(validated_data)
 
-    # Select the basecase forecast interval
-    forecast_input_data = data_with_features[forecast_start:forecast_end]
+    # Only try to make a forecast for moments where load = nan
+    forecast_input = data_with_features.loc[data_with_features["load"].isna(), :]
```

Comment:
What happens if historical load values are also nan due to some error? Should we not enforce a forecast is always for the future?